### PR TITLE
[zh-cn] Translate docs/reference/glossary/feature-gates.md

### PR DESCRIPTION
**Pull request by qlijin**
Translate a new file in the glossary dir: 
[docs/reference/glossary/feature-gates.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/feature-gates.md)

### CODE REVIEW

File: content/zh-cn/docs/reference/glossary/feature-gates.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+<!--
+You can turn these features on or off using the `--feature-gates` command line flag on each Kubernetes component.
+Each Kubernetes component lets you enable or disable a set of feature gates that are relevant to that component.
+The Kubernetes documentation lists all current 
+[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) and what they control.
+-->
+你可以在每个 Kubernetes 组件使用 `--feature-gates` 命令行标志来开启或关闭这些特性。
+每个 Kubernetes 组件都可以让你开启或关闭一组与该组件相关的特性门控。
+Kubernetes 文档列出了所有当前的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/glossary/feature-gates.md:46]:
```suggestion
Kubernetes 文档列出了当前所有的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)及其控制的内容。
```

File: content/zh-cn/docs/reference/glossary/feature-gates.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- fundamental
+- operation
+---
+-->
+
+<!--
+Feature gates are a set of keys (opaque string values) that you can use to control which
+Kubernetes features are enabled in your cluster.
+-->
+特性门控是一组键（opaque 字符串值），你可以用它来控制在你的集群中启用哪些 Kubernetes 特性。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/glossary/feature-gates.md:34]:
```suggestion
特性门控是一组键（不可见的字符串值），你可以用它来控制在你的集群中启用哪些 Kubernetes 特性。
```

**Review by howieyuen — COMMENTED** [review]
/lgtm

**Review by ydFu — APPROVED** [review]
/lgtm

File: content/zh-cn/docs/reference/glossary/feature-gates.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- fundamental
+- operation
+---
+-->
+
+<!--
+Feature gates are a set of keys (opaque string values) that you can use to control which
+Kubernetes features are enabled in your cluster.
+-->
+特性门控是一组键（不可见的字符串值），你可以用它来控制在你的集群中启用哪些 Kubernetes 特性。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/feature-gates.md:34]:
```suggestion
特性门控是一组键（非透明的字符串值），你可以用它来控制在你的集群中启用哪些 Kubernetes 特性。
```

File: content/zh-cn/docs/reference/glossary/feature-gates.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+<!--more-->
+
+<!--
+You can turn these features on or off using the `--feature-gates` command line flag on each Kubernetes component.
+Each Kubernetes component lets you enable or disable a set of feature gates that are relevant to that component.
+The Kubernetes documentation lists all current 
+[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) and what they control.
+-->
+你可以在每个 Kubernetes 组件使用 `--feature-gates` 命令行标志来开启或关闭这些特性。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/glossary/feature-gates.md:44]:
```suggestion
你可以在每个 Kubernetes 组件中使用 `--feature-gates` 命令行标志来开启或关闭这些特性。
```

**Pull request merged**